File: extra05/keith/ex0/Ex0.java
public class Ex0 {
  private static int modadd(int x, int y, int n) {
    System.out.println("in modadd!");
    int res = x + y % n;
    return res;
  }
  public static void main(String[] args) {
    int k = modadd(4,5,7);
    System.out.println(k);
  }
}

File: extra05/keith/ex0/ex0.cpp
#include <iostream>
using namespace std;

int modadd(int x, int y, int n) {
  cout << "in modadd!" << endl;
  int res = x + y % n;
  return res;
}

int main() {
  int k = modadd(4,5,7);
  cout << k << endl;
}
